src/styles/stars.scss

```
@@ -0,0 +1,36 @@
+.stars {
+  display: flex;
+  width: 96px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
```

```suggestion

.&--1 .stars__star:nth-child(-n + 1),
&--2 .stars__star:nth-child(-n + 2),
&--3 .stars__star:nth-child(-n + 3),
&--4 .stars__star:nth-child(-n + 4),
&--5 .stars__star:nth-child(-n + 5) {
```
- Use nesting in scss
- You can combine selectors or you can use a loop to generate them